internal/test/e2e/cleanup.go

```diff
@@ -13,19 +13,29 @@ import (
 	"github.com/aws/eks-anywhere/pkg/validations"
 )
 
-func CleanUpAwsTestResources(storageBucket string, maxAge string, tag string) error {
+const (
+	integrationTestTagKey = "Integration-Test"
+	codebuildIdTagKey     = "CodeBuildId"
+)
+
+func CleanUpAwsTestResources(storageBucket string, maxAge string, tag string, codebuildId string) error {
```

Comment:
What do you think about taking `map[string]string` in the same way you do for `ListInstances`?
I would suggest passing that from the command flags. That way we can clean by any combination of tags and we don't have to keep adding arguments here when we add more tags.